layer/README.md

```diff
@@ -0,0 +1,26 @@
+<!-- markdownlint-disable MD041 MD043-->
+# CDK Powertools layer
+
+This is a CDK project to build and deploy AWS Lambda Powertools layer to multiple commercial regions.
+
+## Build the layer
+
+To build the layer construct you need to provide the powertools version.
+You can pass it as a context variable when running `synth` or `deploy`,
+
+```shell
+cdk synth --context version=1.25.1
+```
+
+## Canary stack
+
+We use a canary stack to verify that the deployment is successful and we can use the layer by adding it to a newly created Lambda function.
+The canary is deployed after the layer construct. Because the layer ARN is created during the deploy we need to pass this information async via SSM parameter.
```

Comment:
```suggestion
The canary is deployed after the Layer construct. Because the Layer ARN is created during the deploy we need to pass this information async via SSM parameter.
```